tests/fast-integration/tracing/client.js

```diff
@@ -1,9 +1,31 @@
 const {
   debug,
   convertAxiosError,
+  retryPromise,
+  waitForVirtualService,
 } = require('../utils');
 const {proxyGrafanaDatasource} = require('../monitoring/client');
 
+const axios = require('axios');
+const https = require('https');
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false, // curl -k
+});
+axios.defaults.httpsAgent = httpsAgent;
+
+async function getTestAppUrl() {
```

Comment:
Let's move everything that is related to calling the application to the telemetry-test and only have the helpers to query Jaeger in the tracing module.